dotCMS/src/test/java/com/dotmarketing/db/DataSourceStrategyProviderTest.java

```diff
@@ -99,30 +106,85 @@ public void testGet(final String testCase)
         Mockito.verify(dbStrategy, Mockito.times(testCase.equals("DBProperties")? 1: 0)).apply();
         Mockito.verify(systemEnvStrategy, Mockito.times(testCase.equals("SystemEnv")? 1: 0)).apply();
         Mockito.verify(dockerSecretStrategy, Mockito.times(testCase.equals("DockerSecret")? 1: 0)).apply();
-        Mockito.verify(tomcatDataSourceStrategy, Mockito.times(testCase.equals("Tomcat")? 1: 0)).apply();
+        Mockito.verify(tomcatDataSourceStrategy, Mockito.times(
+                (testCase.equals("Tomcat") | testCase.equals("CustomProvider")) ? 1 : 0)).apply();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5092420)